README.md

```diff
@@ -75,7 +75,7 @@ appium:intentFlags | Set an optional intent flags to be applied when starting th
 appium:optionalIntentArguments | Set an optional intent arguments to be applied when starting the given appActivity by [Activity Manager](https://developer.android.com/studio/command-line/adb#am)
 appium:dontStopAppOnReset | Set it to `true` if you don't want the application to be restarted if it was already running. `false` by default
 appium:autoLaunch | Whether to launch the application under test automatically (`true`, the default value) after a test starts
-appium:autoGrantPermissions | Whether to grant all the requested application permissions automatically when a test starts(`true`). `false` by default
+appium:autoGrantPermissions | Whether to grant all the requested application permissions automatically when a test starts(`true`). The target SDK version in the application manifest must be greater or equal to 23 and the device under test must be greater or equal to Android 6 (API level 23) to grant permissions. Lower than the target SDK version 22 application must be reisntalled to grant permissions, for example, by settings the `appium:fullRest` capability as `true` for Android 6+ devices. `false` by default
```

Comment:
> device under test

this is rather OS version, not the device version. I'd say `the Android version on the device under test`...

Reply:
by settings > by setting